charts/beta/loki/values.yaml

```diff
@@ -42,12 +48,12 @@ loki:
             period: 24h
     storage:
       type: s3
-      # s3:
-      #   region: us-east-1
-      #   endpoint: s3-fips.us-east-1.amazonaws.com
+      s3:
+        region: us-gov-west-1
       bucketNames:
-        chunks: chunks
-        ruler: ruler
+        # Enter the name(s) of the buckets to use
+        chunks: chunks-bucket-name
+        ruler: ruler-bucket-name
```

Comment:
why are we changing this?

Reply:
Was making it more obvious that the bucket name needed to be added in overrides.  Changed to empty string by default and the README has example/placeholder for how to set.